python/dgl/data/lrgb.py

```diff
@@ -0,0 +1,237 @@
+import hashlib
+import os
+import pickle
+
+import pandas as pd
+from ogb.utils import smiles2graph
```

Comment:
pandas and ogb are not explicit requirements for dgl now. There are two options:
- add them to requirements
- try import them and raise an error containing a pointer to install them if modules not found

What's your opinion? @frozenbugs

Reply:
we now add the try-except in __init__.py like:
``` python
try:
    from .lrgb import PeptidesStructuralDataset
except ImportError:
    pass
```